kzg4844.nimble

```diff
@@ -5,7 +5,7 @@ mode = ScriptMode.Verbose
 ##################################################
 
 packageName   = "kzg4844"
-version       = "0.1.0"
+version       = "1.0.1"
```

Comment:
Oh nice catch. We're actually at v1.0.2 right now though.
```suggestion
version       = "1.0.2"
```

Reply:
oops, forgot to fetch ;)